contribute/style-guides/accessibility.md

```diff
@@ -1,32 +1,28 @@
-# Accessibility at Grafana
+# Style accessibility at Grafana
```

Comment:
I feel like this topic addresses how to develop for accessibility and not style accessibility.